OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -19,9 +19,13 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
+
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[Desc("Distance above CenterPosition for targeting by weapons.")]
```

Comment:
On the other hand, `Capsule` and `Rectangle` shapes can already be positioned along the Y axis, so they're not necessarily centered around the `Actor.CenterPosition`. So admittedly, allowing to fully customize the offset and not just Z would be more thorough and consistent.

I guess I'll give it a try.